code.swift

```diff
@@ -54,25 +54,20 @@ class PeriodicTableViewController: UIViewController {
     }
     
     // This next method was added by a new member of the team. Identify the 10 formatting, naming, and logic issues below:
-    @IBAction func chooseRandomElement(_ sender: Any){
-        let ranElem = elementsContoller.elements.randomElement()
+    @IBAction func chooseRandomElement(_ sender: Any) {
+        let randomElement = elementsContoller.elements.randomElement()
         
-        var chosenElementView:ElementView!
+        var chosenElementView:ElementView?
```

Comment:
The optional doesn’t actually need to change, but there is one more issue here...